src/webfiles/portal/js/juxt.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable */
-var scrollPosition, pjax;
+var scrollPosition, isPostModalOpen, pjax;
```

Comment:
This isn't inherently your fault as the old code does this too, but part of our new standards I'm implementing is defining variables on their own line. This repo just doesn't have our linter properly set up yet, but this linter rule is active in all other repos atm